stackslib/src/net/mod.rs

```diff
@@ -875,6 +875,15 @@ pub struct BlocksData {
     pub blocks: Vec<BlocksDatum>,
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub struct NakamotoBlocksDatum(pub ConsensusHash, pub NakamotoBlock);
```

Comment:
This struct is unnecessary, because Nakamoto blocks contain their tenure's consensus hash already.